From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Urlings <tom.urlings@gmail.com>
Date: Sun, 23 Nov 2025 07:06:18 +0000
Subject: Patching u-boot mvebu files arch/arm/Kconfig

2025-11-23: Tom Urlings: Reapplied build/patch/u-boot/legacy/u-boot-helios4/enable-DT-overlays-support.patch

Signed-off-by: Tom Urlings <tom.urlings@gmail.com>
---
 arch/arm/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 625d2e995d2..731002177a8 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -665,10 +665,12 @@ config ARCH_MVEBU
 	select SPL_DM_SPI if SPL
 	select SPL_DM_SPI_FLASH if SPL
 	select SPL_TIMER if SPL
 	select TIMER if !ARM64
 	select OF_CONTROL
+	select OF_LIBFDT
+	select OF_LIBFDT_OVERLAY
 	select SPI
 	imply CMD_DM
 
 config ARCH_ORION5X
 	bool "Marvell Orion"
-- 
Created with Armbian build tools https://github.com/armbian/build
